packages/ella/src/general/index.ts

```diff
@@ -29,45 +29,39 @@ export { default as isEqual } from 'lodash.isequal';
  */
 export function isEmpty(data: any) {
   try {
-    if (data === null || data === undefined || typeof data === 'undefined') {
```

Comment:
can keep the check for `typeof data === 'undefined'`

https://stackoverflow.com/questions/4725603/variable-undefined-vs-typeof-variable-undefined